lib/cli/src/commands/run/wasi.rs

```diff
@@ -548,10 +551,33 @@ impl Wasi {
         let tokio_task_manager = Arc::new(TokioTaskManager::new(rt_or_handle.into()));
         let mut rt = PluggableRuntime::new(tokio_task_manager.clone());
 
-        if self.networking {
+        let has_networking = self.networking
+            || {
+                if pkg_cache_path.is_file() {
+                    let raw = std::fs::read_to_string(pkg_cache_path)?;
+                    if let Ok(pkg_capability_cache) =
+                        serde_json::from_str::<PkgCapabilityCache>(&raw)
+                    {
+                        tracing::info!("Cache hit for user-set capabilities at path {pkg_cache_path:?}: {pkg_capability_cache:?}");
```

Comment:
nit: maybe `debug` is enough here?

Reply:
[5b8321d](https://github.com/wasmerio/wasmer/pull/4975/commits/5b8321d098b02353e0a960f805851a1456a29338)